docs/source/containerization/kubernetes.mdx

```diff
@@ -1,264 +1,249 @@
-          args:
-            - --hot-reload
-            - --config
-            - /app/configuration.yaml
-          env:
-            - name: APOLLO_KEY
-              valueFrom:
-                secretKeyRef:
-                  name: "release-name-router"
-                  key: managedFederationApiKey
-                  optional: true
-            - name: APOLLO_GRAPH_REF
-              value: REDACTED
-          ports:
-            - name: http
-              containerPort: 80
-              protocol: TCP
-            - name: health
-              containerPort: 8088
-              protocol: TCP
-          lifecycle:
-            {}
-          livenessProbe:
-            httpGet:
-              path: "/health?live"
-              port: 8088
-            initialDelaySeconds: 0
-          readinessProbe:
-            httpGet:
-              path: "/health?ready"
-              port: 8088
-            initialDelaySeconds: 0
-          resources:
-            {}
-          volumeMounts:
-            - name: router-configuration
-              mountPath: /app/
-              readOnly: true
-      volumes:
-        - name: router-configuration
-          configMap:
-            name: release-name-router
-      terminationGracePeriodSeconds: 30
----
-# Source: router/templates/tests/test-connection.yaml
-apiVersion: v1
-kind: Pod
-metadata:
-  name: "release-name-router-test-connection"
-  labels:
-    helm.sh/chart: router-1.31.0
-    app.kubernetes.io/name: router
-    app.kubernetes.io/instance: release-name
-    app.kubernetes.io/version: "v1.31.0"
-    app.kubernetes.io/managed-by: Helm
-  annotations:
-    "helm.sh/hook": test
-spec:
-  containers:
-    - name: netcat
-      image: busybox
-      command: ['nc']
-      args: ['-vz','-w','1','release-name-router:80']
-  restartPolicy: Never
+* Create a YAML file, `my_values.yaml`, to contain additional values that override default values.
+* Edit the file with values to enable metrics endpoints:
+
+  ```yaml title="my_values.yaml"
+  router:
+    configuration:
+      telemetry:
+        metrics:
+          prometheus:
+            enabled: true
+            listen: 0.0.0.0:9090
+            path: "/metrics"
+          otlp:
+            temporality: delta
+            endpoint: <otlp-endpoint-addr>
+  ```
+
+  * `router.telemetry.metrics.prometheus` was already configured but disabled (`enabled: false`) by default. This configuration sets `enabled: true`.
+  * `router.telemetry.metrics.otlp` is enabled by inclusion.
+
+* Deploy the router with the additional YAML configuration file. For example, starting with the `helm install` command from the basic deployment step, append `--values my_values.yaml`:
+
+  ```bash
+  helm install --set managedFederation.apiKey="<graph-api-key>" --set managedFederation.graphRef="<graph-ref>"  oci://ghcr.io/apollographql/helm-charts/router --version <router-version> --values router/values.yaml --namespace <router-namespace> --values my_values.yaml
```

Comment:
@garypen - are the steps here correct, from creating a new values YAML, editing it, and appending it to the `helm install` deployment command? (Those general steps are repeated below for Rhai scripting and the coprocessor)

Reply:
It is correct. I'd just make one cosmetic change to remove the `--namespace` entry from between the two values files. So:
```
helm install --namespace <router-namespace> --set managedFederation.apiKey="<graph-api-key>" --set managedFederation.graphRef="<graph-ref>"  oci://ghcr.io/apollographql/helm-charts/router --version <router-version> --values router/values.yaml --values my_values.yaml
```